### ISSUE TITLE
Wallpapers not loading in list when device language is Arabic

### ISSUE THREAD

**Issue by LinkStatu** [shubham-gupta-16/SimpleWallpaperApp-Android#14]
Hi,
Can you fix this issue please? 
I didn't notice at first when I was using the app. The problem is when I scroll down to see other wallpapers the progress bar continues looping for very long time (10 - 15 min. or more) and the wallpapers don't load. This happens only if the device language is other than English (ex; Arabic, Vietnamese, etc...), like in this **[video](https://streamable.com/f88ju9)**.

When I changed the device language to English, it works fine, like in this **[video](https://streamable.com/6etwxc)**

**Comment by shubham16g** [comment]
First of all, I was busy with some other things, so I didn't give time to the open-source projects. Now I can give time to these projects and issues will be fixed sooner.

**Comment by shubham16g** [comment]
Is that load after 10-15 minutes or just keep showing loader?

**Comment by LinkStatu** [comment]
@shubham-gupta-16 ,
Sometimes loads even after seconds but not for all wallpapers, for example, When I scroll down in the first time, it loads, but after that when I scroll further, the progress bar keeps looping. The time (10-15) is just an estimation by me, cuz I've waited to that time and nothing happened.
Until now, I only noticed Arabic language by testing the app on my device, and Vietnamese by some users who reported the app.
